dev-requirements.txt

```diff
@@ -1,23 +1,18 @@
 # This file is used for testing and development. See
 # https://caremad.io/posts/2013/07/setup-vs-requirement/
 
-# It doesn't seem like pip honors `dependency_links` in setup.py, so install
-# any forked packages here, prior to `-e .`
--e git+https://github.com/jnm/geojson-rewind@0.1.1+py2.jnm#egg=geojson-rewind
-
 # Install this package
 -e .
 
-# Python 2 compatibility
-pyquery<1.4.2
-pyxform<1.0.0
```

Comment:
How does formpack work without pyxform also installed? Is it installed elsewhere?

Reply:
@dorey maybe here?

https://github.com/kobotoolbox/formpack/blob/0863ef337d2cd27990277dcd07844306b115596e/setup.py#L13-L25

Reply:
@dorey, you are right. I must have created my dev environment before running the tests and Pyxform was certainly installed.  
As @joshuaberetta said, I run the test on the fresh install with `tox` which uses the `setup.py` 
I'll fix it.